src/renderer/vt/vtrenderer.hpp

```diff
@@ -204,6 +204,9 @@ namespace Microsoft::Console::Render
 
         bool _WillWriteSingleChar() const;
 
+        // buffer space for these two functions to build their lines
+        // so they don't have to alloc/free in a tight loop
+        std::wstring _bufferLine;

```

Comment:
do we want to til::manage?

Reply:
We discussed this a bit in Teams. This should max out at approximately (`buffer width` * `sizeof(wchar_t)` * 2). We could try to shrink it back down when we start getting small regions, but it would probably grow again when more complicated painting follows up immediately after. The amount of memory left around is arguably small for how much benefit we get from not doing the alloc/dealloc. 

I could come up with some sort of time-based decay of the size of the allocation (if someone was doing big draws and has now been doing little draws for a while, it could release it back.... but I don't really have a justification to do all that to pay back ~240 bytes of memory.)

Also, we only have `til::manage_vector` not `til::manage_string` at this time. So we'd have to write something for that.

We decided together that this is a big MEH.